manifests/conf.pp

```diff
@@ -7,11 +7,12 @@
   include dnsmasq
 
   file { "${dnsmasq::params::config_dir}${prio}-${name}":
-    ensure  => $ensure,
-    owner   => 'root',
-    group   => 0,
-    content => $content,
-    source  => $source,
-    notify  => Class['dnsmasq::service'],
+    ensure       => $ensure,
+    owner        => 'root',
+    group        => 'root',
```

Comment:
Please stick with `0`, in case somebody wants to add support for any *BSD

Reply:
Changed,
Next time I should create a new request instead of attempting to update an old one.
Thanks for your patience.